src/components/pages/login/login.tsx

```diff
@@ -2,19 +2,69 @@ import { Button } from '../../button/button';
 import { Card } from '../../card/card';
 import { Divider } from '../../divider/divider';
 import { Input } from '../../input/input';
+import { useForm } from 'react-hook-form';
 import './Login.css';
+import { validationRegex } from '../../../validation-forms/validation-regex';
+import { errorMessages } from '../../../validation-forms/validation-messages';
```

Comment:
Pensando em aprendizado e melhoria do seu código, dá uma estudada nisso aqui:
https://medium.com/thefork/escaping-relative-import-paths-hell-in-javascript-9f258baeb15e